stylesheets/layout/_nav.scss

```diff
@@ -20,7 +25,12 @@ display: flex;
 flex-direction: row;
 justify-content: flex-start;
 align-items: center;
+margin-top: 20px;
 min-height: 78px;
+
+  &__logo{    
```

Comment:
missing space before {
Multiple times in the code

Reply:
completed